ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+<!--
+If your issue is a usage or a general question, please submit it here instead:
```

Comment:
@tmylk what is a "usage question"? I'd guess only people who use gensim would ever come here and file an issue, so this phrase is confusing.

Can you rephrase this to be clearer / more concrete?